__tests__/app.test.js

```diff
@@ -86,51 +86,135 @@ describe("/api", () => {
+              article_id: expect.any(Number),
+              body: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+            });
           });
-        });
-    });
-    test("GET 200: responds with a single article object with values of article_id property matching the id sent", () => {
-      return request(app)
-        .get("/api/articles/2")
-        .expect(200)
-        .then(({ body }) => {
-          const { article } = body;
-          expect(article.article_id).toBe(2);
-        });
-    });
-    test("GET 404: responds with 404 and msg 'not found' if out of range query sent", () => {
-      return request(app)
-        .get("/api/articles/9999")
-        .expect(404)
-        .then(({ body }) => {
-          expect(body.msg).toBe("not found");
-        });
+      });
+      test("GET 200: responds with a single article object with values of article_id property matching the id sent", () => {
+        return request(app)
+          .get("/api/articles/2")
+          .expect(200)
+          .then(({ body }) => {
+            const { article } = body;
+            expect(article.article_id).toBe(2);
+          });
+      });
+      test("GET 404: responds with 404 and msg 'not found' if out of range query sent", () => {
+        return request(app)
+          .get("/api/articles/9999")
+          .expect(404)
+          .then(({ body }) => {
+            expect(body.msg).toBe("not found");
+          });
+      });
+      test("GET 400: responds with 400 and msg 'bad request' if invalid query type sent", () => {
+        return request(app)
+          .get("/api/articles/invalid_type")
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toBe("bad request");
+          });
+      });
     });
-    test("GET 400: responds with 400 and msg 'bad request' if invalid query type sent", () => {
-      return request(app)
-        .get("/api/articles/invalid_type")
-        .expect(400)
-        .then(({ body }) => {
-          expect(body.msg).toBe("bad request");
-        });
+    describe("PATCH", () => {
+      test("PATCH 200: responds with 200 status and article object when valid request object sent", () => {
+        const patchObj = { inc_votes: 5 };
+        return request(app)
+          .patch("/api/articles/1")
+          .send(patchObj)
+          .expect(200)
+          .then(({ body: { article } }) => {
+            expect(article).toMatchObject({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              body: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+            });
+          });
+      });
```

Comment:
I personally would have combined these tests. I understand why you may have decided to split them, however with the ambiguous expect in your first test (eg expect any votes or article id even though you've been specific in your send) I do feel it is a bit redundant.